frontend/src/libs/schema.js

```diff
@@ -0,0 +1,110 @@
+export const instructorSchema = {
+    keys: ["first_name", "last_name", "utorid", "email"],
+    keyMap: {
```

Comment:
might need some comment here. It is a little bit confusing why there are multiple keys mapping to the same value "last_name", and why some of the keys are strings in double-quotes while some are not.

Reply:
This is from original code. The purpose is to construct a key-mapping between imported JSON keys/ CSV Column names and actual keys of an object like `instructor`. The comment is added. All spreadsheet columns names are strings. I guess the author of this code want to include non-string keys to include situations of JSON keys.